### PR TITLE
[PWD-2489] My Account - Company Info

### PR DESCRIPTION
**Pull request by haikal1223**
### What type of PR is this? (_check all applicable_)

- [x] Feature
- [ ] Bug Fix
- [ ] Documentation Update
- [ ] Setup
- [ ] Code Refactor
- [ ] Performance Improvements
- [ ] Test
- [ ] Build
- [ ] CI
- [ ] Revert

### Screenshots (_optional_)

image

### CODE REVIEW

File: src/views/account/components/AccountWrapper.tsx/CompanyInformationSection.tsx [hunk 1]
```diff
@@ -1,13 +1,144 @@
...
+  const formik = useFormik({
+    initialValues: {
+      namaPerusahaan: '',
+      nibNumber: '',
+      nibFile: null,
+      npwpNumber: '',
+      npwpFile: null
+    },
+    validationSchema,
+    onSubmit: (values) => console.log(values)
```
Comment (shadiqfauzi) [inline src/views/account/components/AccountWrapper.tsx/CompanyInformationSection.tsx:28]: pasang todo untuk handle submit

File: src/views/account/components/AccountWrapper.tsx/CompanyInformationSection.tsx
Comment (katherinedavenia) [inline src/views/account/components/AccountWrapper.tsx/CompanyInformationSection.tsx:1]: mas maap kemaren pas setup gasengaja foldernya tuhh `src/views/account/components/AccountWrapper.tsx/CompanyInformationSection.tsx`, `AccountWrapper.tsx` nya ilangin mass

Comment (haikal1223) [reply src/views/account/components/AccountWrapper.tsx/CompanyInformationSection.tsx:1]: okee kat

**Review by kuntiarso — DISMISSED** [review]

**Review by katherinedavenia — DISMISSED** [review]

**Review by gangsararyop — APPROVED** [review]

File: src/components/PWDKDropzone.tsx [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+        right="8px"
+        width="24px"
+        height="24px"
+        onClick={onDeleteFileClick}
+      >
+        <Close
+          sx={{
+            fontSize: '16px'
+          }}
+        />
```
Comment (Vesals) [inline src/components/PWDKDropzone.tsx:105]:
```suggestion
        <Close sx={{ fontSize: '16px' }} />
```

File: src/components/PWDKDropzone.tsx [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+    }
+    onChange(null)
+  }
+
+  const mappingAccept: Accept = (
+    typeof accept === 'string' ? [accept] : accept
+  ).reduce((result, item) => {
+    result[item] = []
+    return result
+  }, {} as Accept)
```
Comment (Vesals) [inline src/components/PWDKDropzone.tsx:139]: ni udh aman dari warning kmaren kal?

File: src/components/PWDKDropzone.tsx [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+  borderRadius: '6px',
+  backgroundColor: colors.neutral800
+}))
+
+// ------------------------------------------------ components
+
+interface PreviewCardProps {
+  fileName: string
+  fileSize: number
+  fileImage: string | undefined
```
Comment (kuntiarso) [inline src/components/PWDKDropzone.tsx:32]:
```suggestion
  fileImage?: string
```

File: src/components/PWDKDropzone.tsx [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+  onDeleteFileClick: () => void
+}
+
+interface PWDKDropzoneProps {
+  file: File | null
+  accept: string | string[]
+  caption: string
+  maxFileSizeInMb: number
+  error: boolean
+  helperText: string | undefined
```
Comment (kuntiarso) [inline src/components/PWDKDropzone.tsx:42]:
```suggestion
  helperText?: string
```

File: src/components/PWDKDropzone.tsx [hunk 6]
```diff
@@ -0,0 +1,254 @@
...
+  maxFileSizeInMb,
+  error,
+  helperText,
+  onChange,
+  title
+}) => {
+  const inputRef = useRef<HTMLInputElement | null>(null)
+  const [filePreviewUrl, setFilePreviewUrl] = useState<string | undefined>(
+    undefined
+  )
```
Comment (kuntiarso) [inline src/components/PWDKDropzone.tsx:124]:
```suggestion
  const [filePreviewUrl, setFilePreviewUrl] = useState<string>('')
```

bisa pake string kosong sepertinya

File: src/views/account/components/CompanyInformationSection.tsx [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+            maxFileSizeInMb={3}
+            onChange={(file) => {
+              void setFieldValue('npwpFile', file)
+            }}
+            error={Boolean(touched.npwpFile && errors.npwpFile)}
+            helperText={
+              touched.npwpFile && errors.npwpFile
+                ? String(errors.npwpFile)
+                : undefined
+            }
```
Comment (kuntiarso) [inline src/views/account/components/CompanyInformationSection.tsx:132]:
```suggestion
            helperText={touched.npwpFile && String(errors.npwpFile)}
```

begini aja bisa gag

File: src/views/account/components/CompanyInformationSection.tsx [hunk 8]
```diff
@@ -0,0 +1,145 @@
...
+            maxFileSizeInMb={3}
+            onChange={(file) => {
+              void setFieldValue('nibFile', file)
+            }}
+            error={Boolean(touched.nibFile && errors.nibFile)}
+            helperText={
+              touched.nibFile && errors.nibFile
+                ? String(errors.nibFile)
+                : undefined
+            }
```
Comment (kuntiarso) [inline src/views/account/components/CompanyInformationSection.tsx:107]: ini pun

File: src/components/PWDKDropzone.tsx (hunk 3 above)
Comment (haikal1223) [reply src/components/PWDKDropzone.tsx:139]: aman lur

File: src/components/PWDKDropzone.tsx (hunk 6 above)
Comment (haikal1223) [reply src/components/PWDKDropzone.tsx:124]: oiya bisaa

File: src/views/account/components/CompanyInformationSection.tsx (hunk 7 above)
Comment (haikal1223) [reply src/views/account/components/CompanyInformationSection.tsx:132]: bisaa

File: src/views/account/components/CompanyInformationSection.tsx (hunk 8 above)
Comment (haikal1223) [reply src/views/account/components/CompanyInformationSection.tsx:107]: kena ini broodie. Type 'string | false | undefined' is not assignable to type 'string | undefined'.

**Review by kuntiarso — APPROVED** [review]

**Review by Vesals — APPROVED** [review]

**Pull request merged**